apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
+  funcscope: true,
+  newcap: true,
+  nonew: true,
+  shadow: false,
```

Comment:
This option suppresses warnings about variable shadowing i.e. declaring a variable that had been already declared somewhere in the outer scope.  I might have used this incorrectly; "false" is supposed to check local scope only.